Backend/Controller/auth.controller.js

```diff
@@ -0,0 +1,69 @@
+"use strict"
+
+require("dotenv").config();
+const authService = require("../Services/auth.service");
+const userUtils = require("../utils/user.utils");
+const jwt = require("jsonwebtoken");
+
+
+async function registerUser(req, res) {
+  try { 
+
+    const data = await authService.register(req.body);
+    const status = data?.status;
+    const message = data?.message;
+    
+    if (status != 400) {
+      const accesstoken = userUtils.generateToken(req.body.username);
+      res
+        .status(200)
+        .cookie("jwt", accesstoken, {
+          httpOnly: true,
+        })
+        .json({
+          success: true,
+        });
+    } 
+      else{
+        res.status(status).send(message);
+      }
+  } catch (err) {
+    console.log(err);
+    res.status(500).send("An error occurred");
+  }
+}
+
+async function loginUser(req, res) {
+  try {
+    const data = await authService.loginUser(req.body);
+    const status = data?.status;
+    const message=data?.message
```

Comment:
why is this optional?

Reply:
This came because of an accidental merge issue with eslint